uptane-standard.md

```diff
@@ -627,17 +637,17 @@ The inventory database MUST record the following pieces of information:
     * The vehicle identifier the ECU is associated with
     * An ECU key (symmetric or asymmetric; for asymmetric keys, only the public part SHOULD be stored)
     * The ECU key identifier (as defined in {{common_metadata}})
-    * Whether the ECU is a primary or a secondary
+    * A notation of whether the ECU is a Primary or a Secondary
```

Comment:
This isn't just a verbiage change; it changes the requirements.

We had this discussion several months ago: the information that the inventory database needs to have is whether the ECU is a primary or a secondary. That could be indicated by a notation, yes, like a boolean flag. But it could also be indicated by something that is not notational, like which table the ECU is in.

Reply:
Ok. I can put it back. What bothered me was that everything else in the list is a noun (a thing). This one was not. It was a parallelism problem. But if there's no getting around it, I'm change it back. Sorry if its a repeat error. This was the first pass all through the document I've made in months , so I hadn't recalled that discussion.

Reply:
FIXED.